src/action/common/configure_init_service.rs

```diff
@@ -79,10 +79,12 @@ impl ConfigureInitService {
             },
             #[cfg(target_os = "linux")]
             InitSystem::Systemd => {
-                // If /run/systemd/system exists, we can be reasonably sure the machine is booted
-                // with systemd: https://www.freedesktop.org/software/systemd/man/sd_booted.html
-                if !Path::new("/run/systemd/system").exists() {
-                    return Err(Self::error(ActionErrorKind::SystemdMissing));
+                if start_daemon {
```

Comment:
Why do we only check for this path if we're starting the daemon? Should we give the same treatment to checking if `systemctl` exists?

Reply:
In this case that file is only present if systemd is started... and if we're not starting the daemon, there is no need for systemd to be started.

We don't need to give that same treatment to `systemctl` since we still know that the expected init is still indeed `systemd` and we still want to enable it using `systemctl`.

Reply:
Could I ask you to have a brief addition to the comment below (or on the branch itself is fine too) about that (something similar to "/run/systemd/system doesn't exist if systemd is started, which may not be the case in x, y, and z situations, and if we're not starting the daemon, we don't need it" would be great)? Otherwise, lgtm!